### PR TITLE
Windows CI: cli_wait_test

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard jhoward@microsoft.com

This enables a couple more CI tests against a Windows daemon.

### CODE REVIEW

File: integration-cli/docker_cli_wait_test.go [hunk 1]
```diff
@@ -28,6 +27,8 @@ func (s *DockerSuite) TestWaitNonBlockedExitZero(c *check.C) {
 
 // blocking wait with 0 exit code
 func (s *DockerSuite) TestWaitBlockedExitZero(c *check.C) {
+	// Windows busybox does not support trap in this way, not sleep with sub-second
+	// granularity. It will always exit 0x40010004.
```
Comment (cpuguy83) [inline integration-cli/docker_cli_wait_test.go]: This doesn't work on linux either :)
Correct binary is `usleep`

File: integration-cli/docker_cli_wait_test.go [hunk 2]
```diff
@@ -56,11 +57,11 @@ func (s *DockerSuite) TestWaitBlockedExitZero(c *check.C) {
 
 // non-blocking wait with random exit code
 func (s *DockerSuite) TestWaitNonBlockedExitRandom(c *check.C) {
-	testRequires(c, DaemonIsLinux)
+	//testRequires(c, DaemonIsLinux)
```
Comment (cpuguy83) [inline integration-cli/docker_cli_wait_test.go]: Remove

Comment (jhowardmsft) [inline integration-cli/docker_cli_wait_test.go]: Doh. Yes, good catch. Will update in the morning.

Comment (jhowardmsft) [inline integration-cli/docker_cli_wait_test.go]: Fixed

File: integration-cli/docker_cli_wait_test.go (hunk 1 above)
Comment (jhowardmsft) [inline integration-cli/docker_cli_wait_test.go]: Updated.

**Comment by cpuguy83** [comment]
LGTM

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**